app/controllers/application_controller.rb

```diff
@@ -1,3 +1,7 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
+
+  def defualt
```

Comment:
__[staging][Misspell]__

"defualt" is a misspelling of "default"


<!-- Comment by [Sider](https://sider.review) -->